demisto_sdk/commands/update_release_notes/update_rn.py

```diff
@@ -169,10 +169,10 @@ def handle_existing_rn_version_path(self, rn_path: str) -> str:
                 The content of the rn
         """
         if self.existing_rn_version_path:
-            self.should_delete_existing_rn = self.existing_rn_version_path != rn_path
+            existing_rn_abs_path = Path(self.existing_rn_version_path).absolute()
+            self.should_delete_existing_rn = str(existing_rn_abs_path) != rn_path
```

Comment:
the safest thing would be comparing both when they're absolute, i.e. `Path(self.existing_rn_version_path).absolute() == Path(rn_path).absolute()`